packages/connectivity/example/ios/Runner/Info.plist

```diff
@@ -22,6 +22,10 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+	<string>Location service</string>
```

Comment:
I think you should put an appropriate English sentence here since it's an example app.